nautobot/ipam/utils/__init__.py

```diff
@@ -46,6 +47,12 @@ def add_available_ipaddresses(prefix, ipaddress_list, is_pool=False):
             )
         ]
 
+    # sort the IP address list
```

Comment:
Added some logic to fix the `add_available_ipaddresses` method that was expecting a sorted list of IP Addresses (undocumented). Annotating the queryset passed in to this method broke the default sort and was causing exceptions to be raised below.

Reply:
maybe elaborate the code comment to document this?